source/scripts/app.js

```diff
@@ -650,6 +650,14 @@ async function populateCookbooksPage() {
   for (const cookbook of cookbooks) {
     let card = document.createElement("cookbook-card");
     card.cookbook = cookbook;
+
+    let shadow = card.shadowRoot;
+    let title = shadow.querySelector(".title").innerHTML;
+    if (title === "My cookbook") {
+      let button = shadow.getElementById("remove");
+      button.remove();
+    }
```

Comment:
Could you populate the default cookbook at the first cookbook? You can use [prepend](https://developer.mozilla.org/en-US/docs/Web/API/Element/prepend) to insert an element as the first child

Reply:
I think the first cookbook will always be the default because it's created before any other cookbook is created, so the default should be at index 0 of the array returned by the getAllCookbooks function. I'm not sure if this covers what you're talking about.

Reply:
Oh, it's because IndexedDb sorts string keys alphabetically. So, when you retrieve all cookbooks, they come back in alphabetical order when you iterate through them.